src/scripts/validate.js

```diff
@@ -18,6 +18,7 @@ const useDefaultScripts = typeof validateScripts !== 'string'
 const scripts = useDefaultScripts
   ? {
       build: ifScript('build', 'npm run build --silent'),
+      format: precommit ? null : ifScript('format', 'npm run format --list-different'),
```

Comment:
This should be: `npm run format -- --list-different` otherwise the `--list-different` arg will be passed to npm.

Thanks!

Reply:
![yarn is not npm](http://www.addletters.com/pictures/bart-simpson-generator/bart-simpson-generator.php?line=yarn+is+not+npm)